libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -78,6 +79,7 @@
     private String mGoogleEmail;
     private String mRequestedEmail;
     private boolean mIsSocialLogin;
+    private boolean mIsValidEmail = false;
```

Comment:
The default value of a `boolean` is already `false` so you don't have to specify 😄